File: Dataset/cpp/messages.cpp

//  messages.cpp (c) 1998-2002 Kari Laitinen

#include  <iostream.h>

void  print_message()
{
   cout << "\n   This is function named \"print_message\"." ;
   cout << "\n   Functions usually contain many statements. " ;
   cout << "\n   Let us now return to the calling program." ;
}

int main()
{
   cout << "\n THIS IS THE FIRST STATEMENT IN FUNCTION \"main\"." ;

   print_message() ;

   cout << "\n THIS IS BETWEEN TWO FUNCTION CALLS." ;

   print_message() ;

   cout << "\n END OF FUNCTION \"main\".\n" ;
}

File: Dataset/cs/Messages.cs

//  Messages.cs  (c) 2004 Kari Laitinen

//  www.naturalprogramming.com

using System ;

class Messages
{
   static void print_message()
   {
      Console.Write( "\n   This is method named \"print_message()\"." ) ;
      Console.Write( "\n   Methods usually contain many statements. " ) ;
      Console.Write( "\n   Let us now return to the calling method." ) ;
   }

   static void Main()
   {
      Console.Write( "\n THE FIRST STATEMENT IN METHOD \"Main()\"." ) ;

      print_message() ;

      Console.Write( "\n THIS IS BETWEEN TWO METHOD CALLS." ) ;

      print_message() ;

      Console.Write( "\n END OF METHOD \"Main()\".\n" ) ;
   }
}



